app/grants/templates/grants/detail/tabs.html

```diff
@@ -14,6 +14,7 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
+{% load grants_extra %}　
```

Comment:
Could we remove this? It's already being loaded in the line below

Reply:
yeah, I did not notice that.